components/SingleArticle.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect } from "react";
+import { StyleSheet, Text, View, Image, Dimensions } from "react-native";
+import { useSelector } from "react-redux";
+import ArticleServices from "../modules/ArticleServices";
+
+const SingleArticle = (props) => {
+  const { id } = props.route.params.article;
+  const { currentArticle } = useSelector((state) => state);
```

Comment:
Same here, remove the parenthesis around (state)

Reply:
done :)